app/models/comment.rb

```diff
@@ -163,6 +164,37 @@ def root_exists?
     ancestry && Comment.exists?(id: ancestry)
   end
 
+  def edited(new_attributes, by:)
```

Comment:
This conflicts with the `edited` boolean attribute defined on the `comments` table. Maybe we can call it `log_edit` or `create_edit` or similar.